2-classes-db/task.md

```diff
@@ -29,22 +35,45 @@ To submit this homework write the correct commands after each question.
 ### 3. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'Fee'.
 ```sql
 
+select date, transaction_no, supplier_inv_no, description, amount
+from spends
+where description like '%fee%';
+
 ```
 ### 4. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'fee' (case insensitive). You will need to search 'https://www.postgresql.org/docs/' to solve this.
 ```sql
 
+select date, transaction_no, supplier_inv_no, description, amount
+from spends
+where description like '%Fee%';
+
 ```
 ### 5. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose amount is £25,000, £30,000, £35,000 or £40,000.
 ```sql
 
+select date, transaction_no, supplier_inv_no, description, amount
+from spends
+where amount in('25000','30000','35000', '40000');
+
 ```
 ### 6. Show the date, the supplier_id, the description and the amount for transactions with the expense area of 'Better Hospital Food'. You could do a query to get the expense_area_id first then do a query to find the dates, supplier_ids and amounts. But it would be better to do this all in one query by linking the tables together using INNER JOINs.
 ```sql
 
+select date, supplier_id, description, amount
+from spends
+inner join expense_areas  
+on spends.expense_area_id = expense_areas.id
+where expense_areas.expense_area like '%Better Hospital Food%';
+
 ```
 ### 7. Show the date, supplier name, description and amount for transactions with the expense area of 'Better Hospital Food'. You will need to INNER JOIN another table to be able to do this.
 ```sql
 
+select spends.date, suppliers.supplier, spends.description, spends.amount
+from spends
+inner join suppliers 
+on spends.supplier_id = suppliers.id;
+
```

Comment:
You need to only select transactions with the expense area "Better Hospital Food" - so combine what you have done in Q 6 with this.